lib/ReactViews/Preview/MappablePreview.jsx

```diff
@@ -39,12 +39,16 @@ class MappablePreview extends React.Component {
     t: PropTypes.func.isRequired
   };
 
-  async toggleOnMap(event) {
+  async toggleOnMap(_event) {
     if (defined(this.props.viewState.storyShown)) {
       runInAction(() => (this.props.viewState.storyShown = false));
     }
 
-    const keepCatalogOpen = event.shiftKey || event.ctrlKey;
+    const keepCatalogOpen =
+      this.props.viewState.terria.configParameters.keepCatalogOpen ||
+      event.shiftKey ||
+      event.ctrlKey;
+    event.shiftKey || event.ctrlKey;
```

Comment:
Is this line an expression without any side effects? Should probably be removed in that case?